pkg/storaged/utils.js

```diff
@@ -659,6 +665,19 @@ export function is_netdev(client, path) {
     return false;
 }
 
+export function should_ignore(client, path) {
+    if (!client.in_anaconda_mode())
+        return false;
+
+    const parents = get_direct_parent_blocks(client, path);
+    if (parents.length == 0) {
+        const b = client.blocks[path];
+        return b && client.should_ignore_block(b);
+    } else {
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19352-20231205-084155-675f6df4-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)